generate_bindings.sh

```diff
@@ -9,10 +11,22 @@ echo '#include "fmgr.h"' >> /tmp/postgres.c
 echo '#include "replication/output_plugin.h"' >> /tmp/postgres.c
 echo '#include "replication/logical.h"' >> /tmp/postgres.c
 
+gcc -I /usr/local/Cellar/postgresql/9.5.0/ \
```

Comment:
This stuff can all be gathered from `pg_config`; that would seem to be a better investment of effort.